src/main/scala/org/zalando/spearheads/innkeeper/services/RouteToEskipTransformer.scala

```diff
@@ -54,12 +54,19 @@ class DefaultRouteToEskipTransformer @Inject() (hostsService: HostsService, comm
     NameWithStringArgs("Host", Seq(hostsRegex))
   }
 
-  private[this] def createPathPredicate(pathUri: String, hasStar: Boolean) = {
-    val starSuffix = if (hasStar) "/**" else ""
+  private val STAR_PATH_SUFFIX = "/**"
+  private[this] def createPathPredicate(pathUri: String, hasStar: Boolean, isRegex: Boolean) = {
+    val pathArg = (hasStar, isRegex) match {
+      case (false, false) => wrapInQuotes(pathUri)
+      case (true, false)  => wrapInQuotes(pathUri + STAR_PATH_SUFFIX)
+      case (_, true)      => pathUri
```

Comment:
I made the assumption that the slashes are in the string. I guess it is better to only have the regex expression. I'll change it.